ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -960,6 +967,7 @@ private[platform] object JdbcLedgerDao {
       metrics: Metrics,
       lfValueTranslationCache: LfValueTranslation.Cache,
       validatePartyAllocation: Boolean = false,
+      jdbcAsyncCommits: Boolean = false,
```

Comment:
If this only needs to be false for DAML on SQL, why not default it to `true`?

(Or, better yet, don't have a default?)

Reply:
You're right. Not default then.